app/bundles/PluginBundle/Entity/IntegrationEntityRepository.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\PluginBundle\Entity;
+
+use Mautic\CoreBundle\Entity\CommonRepository;
+
+/**
+ * IntegrationRepository
+ */
+class IntegrationEntityRepository extends CommonRepository
+{
+
+    public function getIntegrationsEntityId($integration, $integrationEntity, $internalEntity, $internalEntityId)
```

Comment:
`$internalEntityId` is required but there is at least one use of getIntegrationsEntityId() that isn't passing in a fourth argument. Was `$internalEntityId` meant to be defaulted to null?